src/metabase/api/util.clj

```diff
@@ -53,4 +54,13 @@
         headers   {"Content-Disposition" "attachment; filename=\"connection_pool_info.json\""}]
     (assoc (response/response {:connection-pools pool-info}) :headers headers, :status 200)))
 
+
+(api/defendpoint POST "/product-feedback"
+  "Send feedback about metabase"
+  [:as {{:keys [comments source email]} :body}]
+  {comments ms/NonBlankString
+   source ms/NonBlankString
+   email [:maybe :string]}
```

Comment:
`[:maybe ms/NonBlankString]` maybe?

Reply:
Do we have a strong preference for null over empty string? Is it because of how they're handled in the db?